pkg/services/featuremgmt/registry.go

```diff
@@ -1254,6 +1254,13 @@ var (
 			Stage:       FeatureStageExperimental,
 			Owner:       grafanaFrontendPlatformSquad,
 		},
+		{
+			Name:        "timeRangePan",
+			Description: "Enables time range panning functionality",
+			Stage:       FeatureStageExperimental,
+			Owner:       grafanaDatavizSquad,
+			Expression:  "false",
```

Comment:
I could be wrong so please verify and sorry for not noticing this earlier, but I don't think you need `Expression: "false"` for `FeatureStageExperimental`, only for `FeatureStageGeneralAvailability`. I've also used `FrontendOnly: true,` in the past.

Reply:
Yeah, just confirmed that for experimental features, expression is not required as a value of true is not allowed for experimental features.